app/templates/index.html

```diff
@@ -5,34 +5,21 @@
 {% endblock %}
 {% block content %}
 
-<div class="grid-row main-wrapper--lg">
-	<div class="grid-column-full">
-		<h2>Tasks</h2>
-		<div class="page--content">
-			<p>Pick a task length.</p>
-			<div class="button-group" role="group" aria-label="Basic example">
-				<button type="button" class="button" onclick="handleClick('small')">50 participants</button>
-				<button type="button" class="button" onclick="handleClick('large')">500 participants</button>
-			</div>
-		</div>
-	</div>
+<div class="grid-row">
+  <div class="grid-column-two-thirds">
+    <div class="page--content">
+    <h2 class="heading-md">Before you start</h2>
+    <ol>
+      <li><strong>This tool will only work with data in specific formats.</strong> You must have two files – <code>mentees.csv</code> and <code>mentors.csv</code> – to start the matching process. Those files need to be formatted with specific column headings. You can <a href="#">download a template</a> from this website.</li>
```

Comment:
I'll add an issue to the backlog to create a template for this, otherwise I'll forget

Reply:
In fact I think I already did this! https://github.com/jonodrew/mentor-match-package/blob/main/templates/participant.csv